projects/storefrontlib/layout/launch-dialog/config/launch-config.ts

```diff
@@ -90,4 +90,5 @@ export const enum LAUNCH_CALLER {
   ANONYMOUS_CONSENT = 'ANONYMOUS_CONSENT',
   REPLENISHMENT_ORDER = 'REPLENISHMENT_ORDER',
   PLACE_ORDER_SPINNER = 'PLACE_ORDER_SPINNER',
+  ADDED_TO_CART_TOAST = 'ADDED_TO_CART_TOAST',
```

Comment:
We should use type augmentation and declare this value in the cart lib.
Search for `const enum LAUNCH_CALLER` for examples.